### PR TITLE
Add optional icon to form field description

### PR DESCRIPTION
**Pull request by lk316**
# Description

Add an optional icon to the form field description. 

## Issue ticket number and link

https://trello.com/c/VNxm1wW4/157-pcbs-edit-profile

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: public/views/partials/molecules/formfield.liquid [hunk 1]
```diff
@@ -49,6 +50,7 @@ metadata:
         content: Form label
       description_params:
         content: This is a FormField description
+        icon: Name
```
Comment (ogabor82) [inline public/views/partials/molecules/formfield.liquid:53]: In the Styleguide this would be an exact name of an icon.

Comment (Slashek) [reply public/views/partials/molecules/formfield.liquid:53]: @ogabor82 could you please use github's suggestion feature to suggest a change? thanks!

Comment (ogabor82) [inline public/views/partials/molecules/formfield.liquid:53]:
```suggestion
        icon: 'info'
```

Comment (lk316) [reply public/views/partials/molecules/formfield.liquid]: changed

Comment (lk316) [reply public/views/partials/molecules/formfield.liquid]: changed

**Review by ogabor82 — APPROVED** [review]

**Pull request merged**